docker/build.sh

```diff
@@ -18,11 +18,11 @@
 # under the License.
 
 #
-# Execute command within a docker container
+# Build a docker container and optionally execute command within a it
 #
 # Usage: build.sh <CONTAINER_TYPE> [--tag <DOCKER_IMAGE_TAG>]
 #                [--dockerfile <DOCKERFILE_PATH>] [-it]
-#                [--net=host] [--cache-from <IMAGE_NAME>] <COMMAND>
+#                [--net=host] [--cache-from <IMAGE_NAME>] <COMMAND (optional)>
```

Comment:
could you enclose command in `[]` to mark it optional, like the other options here? `[<COMMAND>]`

Reply:
done